examples/seismic/acoustic/acoustic_example.py

```diff
@@ -49,17 +49,17 @@ def acoustic_setup(shape=(50, 50, 50), spacing=(15.0, 15.0, 15.0),
 
     # Create solver object to provide relevant operators
     solver = AcousticWaveSolver(model, source=src, receiver=rec,
-                                time_order=time_order,
+                                kernel=kernel,
```

Comment:
can we avoid one line here? It's OK in a next PR

Reply:
Probaly, let e update it so